app/pages/home/apply.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { RoundedButton } from '../../components';
+import { SectionHeader, SectionBody } from './section_components.jsx';
+
+class Apply extends React.Component {
+    render() {
+        return (
+            <div>
+                <SectionHeader>Apply</SectionHeader>
+                <SectionBody>Apply to MHacks Nano! We will just ask you a couple questions about yourself, your hackathon experiences and allow you to form a team.</SectionBody>
+                <a href="/login">
```

Comment:
If you look in the Header component you can find reference to NavLink with routes.* which you can use to navigate around